### PR TITLE
CSS Transitions

### PR DESCRIPTION
**Pull request by sfernandez11**
# Summary
## Features
*Added CSS Transitions to the tech boxes
## Screenshots
tech

### CODE REVIEW

File: scss/techs.scss [hunk 1]
```diff
@@ -22,6 +28,8 @@ $card-width: 210px;
     left: -10px;
     position: absolute;
     top: -10px;
+    transition: left 0.5s,
```
Comment (frparodi) [inline scss/techs.scss]: Arrange both properties on the same line.

File: scss/techs.scss [hunk 2]
```diff
@@ -12,6 +12,12 @@ $card-width: 210px;
 .main-tech-container {
   position: relative;
   margin-right: 30px;
+  &:hover {
```
Comment (frparodi) [inline scss/techs.scss]: Leave an empty line before hover.

File: scss/techs.scss [hunk 3]
```diff
@@ -12,6 +12,12 @@ $card-width: 210px;
...
+    .background-tech-container {
+      left: 10px;
+      top: 10px;
+    }
+  }
 }
 
 .background-tech-container {
```
Comment (frparodi) [inline scss/techs.scss]: I forgot to mention you have to sort class properties alphabetically. You can install a plugin that highlights those properties that are not sorted. I don't pretend you modify all code, but keep it in mind :smile:

File: scss/techs.scss (hunk 1 above)
Comment (sfernandez11) [reply scss/techs.scss]: Done

File: scss/techs.scss (hunk 2 above)
Comment (sfernandez11) [reply scss/techs.scss]: Done

**Pull request merged**